### PR TITLE
Log collection example for sidecar containers

### PR DESCRIPTION
**Pull request by seddarj**

### CODE REVIEW

File: 16-sidecar-containers/README.md [hunk 1]
```diff
@@ -33,14 +33,20 @@ Istio, the service mesh tool, installs a sidecar container to do its job: https:
...
 ## Exercices
 
-Review and apply the file [01-sidecar.yml](01-sidecar.yml). Connect to the `nginx` container and look at the file system in `/usr/share/nginx/html`.
+Review and apply the file [01-sidecar.yml](01-sidecar.yml).
+
+Connect to the `nginx` container (`kubectl exec -it two-containers -c nginx`) and look at the file system in `/var/log/nginx`.
+
+Do the same for the `fluentd` container and this time look in `/logs`. What do you see?
+
+Tail the logs from the fluentd pod (`kubectl logs -f two-containers -c fluentd`) and in another terminal window, send requests to the nginx service with a curl from a separate pod. What do you see? How do you explain it?
```
Comment (Jerska) [inline 16-sidecar-containers/README.md:42]: `nginx`'s image appears to have `curl` installed.
I wanted to run it from the `fluentd` image, but it doesn't have it. :/

```suggestion
Tail the logs from the fluentd pod (`kubectl logs -f two-containers -c fluentd  -- /bin/sh`) and in another terminal window, send requests to the nginx service with a `curl` from a separate pod (or nginx's pod itself). What do you see? How do you explain it?
```

File: 16-sidecar-containers/README.md [hunk 2]
```diff
@@ -33,14 +33,20 @@ Istio, the service mesh tool, installs a sidecar container to do its job: https:
 
 ## Exercices
 
-Review and apply the file [01-sidecar.yml](01-sidecar.yml). Connect to the `nginx` container and look at the file system in `/usr/share/nginx/html`.
+Review and apply the file [01-sidecar.yml](01-sidecar.yml).
+
+Connect to the `nginx` container (`kubectl exec -it two-containers -c nginx`) and look at the file system in `/var/log/nginx`.
```
Comment (Jerska) [inline 16-sidecar-containers/README.md:38]:
```suggestion
Connect to the `nginx` container (`kubectl exec -it two-containers -c nginx -- /bin/sh`) and look at the file system in `/var/log/nginx`.
```

**Review by Jerska — APPROVED** [review]
Super clear! It's a great way to see the common underlying volume.

File: 16-sidecar-containers/README.md (hunk 1 above)
Comment (seddarj) [reply 16-sidecar-containers/README.md:42]: I agree with the second part but not the first one (the `-- /bin/sh`), I'm tailing the logs here 😅

**Comment by seddarj** [comment]
Thanks for the review and the suggestions @Jerska!

File: 16-sidecar-containers/README.md (hunk 1 above)
Comment (Jerska) [reply 16-sidecar-containers/README.md]: Ha yes indeed! I called `tail` from within the container, but indeed I could have called `tail` directly from the `exec` command. :)

**Pull request merged**